packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1420,10 +1413,10 @@ export const audiusBackend = ({
     return libs.Account!.resetPassword({ username, password })
   }
 
-  async function sendRecoveryEmail() {
+  async function sendRecoveryEmail(handle: string) {
     await waitForLibsInit()
     const host = getHostUrl()
-    return audiusLibs.Account.generateRecoveryLink({ host })
+    return audiusLibs.Account.generateRecoveryLink({ handle, host })
```

Comment:
what happens if i pass someone else's handle in here?

Reply:
You're not going to believe this, but we don't even use the `handle` in identity... 🤦 
If I'm not mistaken, we can remove this entirely.